packages/xod-cli/README.md

```diff
@@ -9,23 +9,265 @@ other packages. The responsibility of `xod-cli` is to parse command line
+      	second position is the patch name like 'main' or '@/main'
+
+OPTIONS
+  -h, --help                 show CLI help
+  -o, --output=output
+  -q, --quiet                do not log messages other than errors
+  -v, --version              show CLI version
+  -w, --workspace=workspace  [default: /home/sk/xod] use the workspace specified, defaults to $HOME/xod
+
+ALIASES
+  $ xodc r
+
+EXAMPLES
+  Exports the current multifile project to a xodball
+  $ xodc resave . -o ~/foo.xodball
+
+  Outputs the current multifile project as a xodball to stdout
+  $ xodc resave
+
+  Resaves one xodball into another (useful for applying migrations)
+  $ xodc resave foo.xodball -o bar.xodball
+
+  Converts a xodball to a multifile project
+  $ xodc resave foo.xodball -o /some/new/dir
+```
+
+_See code: [src/commands/resave.js](https://github.com/xodio/xod/blob/v0.25.0/packages/xod-cli/src/commands/resave.js)_
+
+## `xodc tabtest [options] [project]`
+
+tabtest project
+
+```
+USAGE
+  $ xodc tabtest [options] [project]
+
+ARGUMENTS
+  ENTRYPOINT
+      multipositional argument, reference to project and patch:
+      if omitted:
+      	if current working directory is a project root, then use this project and @/main patch
+      	if current working directory is a patch directory, then use this project and this patch
+      if one position:
+      	if path to a patch directory or file, then use this project and this patch
+      	if path to a project/xodball, then use this project and @/main patch
+      if two positions:
+      	first position is the path to a project/xodball
+      	second position is the patch name like 'main' or '@/main'
+
+OPTIONS
+  -h, --help                   show CLI help
+  -o, --output-dir=output-dir  [default: /tmp/xod-tabtest] path to directory where to save tabtest data
+  -q, --quiet                  do not log messages other than errors
+  -v, --version                show CLI version
+  -w, --workspace=workspace    [default: /home/sk/xod] use the workspace specified, defaults to $HOME/xod
+  --no-build                   do not build
+```
+
+_See code: [src/commands/tabtest.js](https://github.com/xodio/xod/blob/v0.25.0/packages/xod-cli/src/commands/tabtest.js)_
+
+## `xodc transpile [options] [entrypoint]`
+
+transpile (generate C++) a XOD program
+
+```
+USAGE
+  $ xodc transpile [options] [entrypoint]
+
+ARGUMENTS
+  ENTRYPOINT
+      multipositional argument, reference to project and patch:
+      if omitted:
+      	if current working directory is a project root, then use this project and @/main patch
+      	if current working directory is a patch directory, then use this project and this patch
+      if one position:
+      	if path to a patch directory or file, then use this project and this patch
+      	if path to a project/xodball, then use this project and @/main patch
+      if two positions:
+      	first position is the path to a project/xodball
+      	second position is the patch name like 'main' or '@/main'
```

Comment:
This description is too formal and hard to understand IMO. Suggest a less precise, but simpler alternative:

```
ENTRYPOINT
  Project and/or patch to operate on. The project should point to a file or directory on the
  file system. The patch may either point to file system or be a XOD patch path. If either
  is omitted, it is inferred from the current working directory or another argument. Examples:

    * ./path/to/proj.xodball main      # xodball + patch name
    * ./path/to/proj/main/patch.xodp   # just full path to a patch
    * main                             # a patch in the current project
```